CIP-1694/README.md

```diff
@@ -0,0 +1,664 @@
+
+* **Governance Action Type**<br/>
+  The type of governance action. Note that the three possible treasury actions involve Lovelace amounts $T_0$, $T_1$, $T_2$, and $T_3$.
+
+* **Constitutional Committee**<br/>
+  A value of :heavy_check_mark: indicates that `Quorum-Many` Constitutional Committee 'yes' votes are required.<br/>
+  A value of :x: means that Constitutional Committee votes do not apply.
+
+* **DReps**<br/>
+  The DRep vote threshold that must be met as a percentage of *active voting stake*, ranging from 0 to 100 (inclusive).
+
+* **AVST**<br/>
+  The _**A**ctive **V**oting **S**take **T**hreshold_. The percentage to be used to determine if there is *sufficient active voting stake*.
+
+* **AVST fallback**<br/>
+  The fallback condition if the AVST threshold is not met
+  * **None**: there is no fallback, the action cannot be ratified unless the AVST threshold is met
+  * **SPO vote**: the action can be ratified if there is a sufficient SPO vote endorsement.
+
+
+* **SPOs**<br/>
+  The SPO vote threshold which must be met as a percentage of the stake held by all stake pools. The SPO vote is only considered if the AVST threshold is :x: or the AVST is below the AVST threshold.
+
+| Governance Action Type                               | Constitutional Committee | DReps    | AVST     | AVST Fallback | SPOs     |
+| ---                                                  | :---:                    | ---      | ---      | ---           | ---      |
+| 1. Motion of no-confidence                           | :x:                      | $P_1$    | $Q_1$    | None          | $R_1$    |
+| 2(a). New Committee/quorum (_normal state_)          | :heavy_check_mark:       | $P_{2a}$ | $Q_{2a}$ | SPO Vote      | $R_{2a}$ |
+| 2(b). New Committee/quorum (_state of no-confidence_)| :x:                      | $P_{2b}$ | $Q_{2b}$ | None          | $R_{2b}$ |
+| 3. Update to the Constitution                        | :heavy_check_mark:       | $P_3$    | $Q_3$    | SPO Vote      | $R_3$    |
+| 4. Hard-Fork initiation                              | :heavy_check_mark:       | $P_4$    | 0        | -             | $R_4$    |
+| 5. Protocol parameter changes                        | :heavy_check_mark:       | $P_5$    | $Q_5$    | SPO Vote      | $R_5$    |
+| 6(a). Treasury withdrawal, $[T_0, T_1)$              | :heavy_check_mark:       | $P_{6a}$ | $Q_{6a}$ | SPO Vote      | $R_{6a}$ |
+| 6(b). Treasury withdrawal, $[T_1, T_2)$              | :heavy_check_mark:       | $P_{6b}$ | $Q_{6b}$ | SPO Vote      | $R_{6b}$ |
+| 6(c). Treasury withdrawal, $[T_2, T_3)$              | :heavy_check_mark:       | $P_{6c}$ | $Q_{6c}$ | SPO Vote      | $R_{6c}$ |
+
+Some of the parameters given in this table ( $P_1$ ... $R_{6c}$, $T_1$ ... $T_3$ ) may be updatable protocol parameters, but others should be hard-coded. This proposal deliberately leaves both this choice and the choice of actual parameter values open for discussion.
```

Comment:
I'd like to point out that the only difference between a hard-coded value and a protocol parameter is what mechanism can be used to change them. It's always possible to change a hard-coded value via a hard fork, which isn't even strictly more difficult because there's no AVST requirement.

With that in mind, and considering that some protocol parameters need to be tuned more carefully than others (messing with `maxHeaderSize` is much more dangerous than messing with fees), I think we should consider a tiered system to protocol parameter changes (similar to treasury withdrawals) and simply have all of those values be protocol parameters.